### PR TITLE
fix: ensure correct run-context for pg instrumentation

### PR DESCRIPTION
**Pull request by trentm**
Refs: #2430

### Checklist

- [x] Implement code
- [x] Add tests
- [x] Add CHANGELOG.asciidoc entry
- [x] Commit message follows [commit guidelines](https://github.com/elastic/apm-agent-nodejs/blob/master/CONTRIBUTING.md#commit-message-guidelines)
- [x] TAV run

### CODE REVIEW

**Comment by trentm** [comment]
run module tests for pg

File: lib/instrumentation/modules/pg.js [hunk 1]
```diff
@@ -10,12 +10,15 @@ var symbols = require('../../symbols')
 var { getDBDestination } = require('../context')
 
 module.exports = function (pg, agent, { version, enabled }) {
+  if (!enabled) {
```
Comment (astorm) [inline lib/instrumentation/modules/pg.js:13]: +1 -- this is both more correct and should not effect the functionality of the instrumentation -- when `enabled` was passed on to the `patchClient` method it bailed in a guard clause immediately.

**Review by astorm — APPROVED** [review]
+1 -- looks like an uneventful swapping in of the new context handling.

**Comment by trentm** [comment]
run module tests for pg

**Pull request merged**